app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -87,6 +87,8 @@
             android:layout_height="wrap_content"
             android:layout_marginTop="16dp"
             android:hint="@string/add_profile_input_name"
+            app:expandedHintEnabled="false"
+            app:suffixText="*"
```

Comment:
We don't actually need to use the asterisk anymore. The main adjustment is that we can remove the "* Required" label from the screen and instead add "Required" as suffix text to fields whose values are required.

Reply:
@BenHenning  Noted, however, on the same page, we have 2 more edit boxes, that have the * design. Should I go ahead and also change(update) them? Or should I focus on what's mentioned in the issue?

Just to be clear, should talkback announce the required?

Reply:
@BenHenning, what I've done.

1. Remove text field *Required in XML.
2. Remove asterisks from the VM variable.
3. Rename the variable to requiredField.
4. Remove the associated test, since it's not required anymore.
5. Use the VM variable to have localization when required called in the edit text fields(helper and suffix text localization).
6. remove * from all string files associated with this issue., in all languages.
7. Add helper text.

| sighted | not sighted |
| ------------- | ------------- |
[visual.webm](https://user-images.githubusercontent.com/31986629/195029823-a619c019-89db-47b6-ba98-db63122bceb1.webm) | [notvisually.webm](https://user-images.githubusercontent.com/31986629/195029920-f5309a8b-566a-47cd-968a-468ec95d7b7e.webm)

Reply:
Thanks @Ryggs. Can you:
1. Record the video via the 'logcat' tab in Android Studio? That will produce a mp4 that you can upload to GitHub such that it will be able to be played directly within the thread.
2. Go into the developer options for Talkback (not the Android system) and enable "Display speech output", then re-record the video? This will make it easier to follow along than listening to the audio (since there seems to be a delay in the recording, and the timing is actually really important in understanding what's going on here since there are multiple "Required" lines to read out).